SPECS/util-linux/util-linux.spec

```diff
@@ -11,6 +11,7 @@ Source0:        https://mirrors.edge.kernel.org/pub/linux/utils/%{name}/v2.37/%{
 Source1:        runuser
 Source2:        runuser-l
 BuildRequires:  audit-devel
+BuildRequires:  libcap-ng-devel
```

Comment:
Since it's a toolchain package, this alone might not work, because `libcap-ng` is not part of the toolchain. We don't rebuild toolchain packages during regular package build phase, so `setpriv` will probably still not be there. You could either add `libcap-ng` (and its dependencies) to the toolchain or move `util-linux` out of the toolchain (probably not possible). There might be another, better solution, these are simply the ones, which came to my head right now.

Reply:
Looks like `libcap-ng` depends only on `python3-devel` and `swig` to build. Both are already in the toolchain, so adding `libcap-ng` there as well should be fairly simple. I'd double-check with Andrew and Olivia, if there might still be a better way, though.